test_http.py

```diff
@@ -0,0 +1,36 @@
+from echo_server import create_server_socket, response_ok, response_error, parse_request
+import pytest
+
+
+def test_ok():
+    assert response_ok() == """\
+HTTP/1.1 200 OK\r\n\
+Content-Type: text/plain\r\n\
+\r\n\
+Hey you.\r\n\
+"""
+
```

Comment:
tough to write a good test for such a simple function.  Maybe just assert that 200 and "OK" are in the first line?  Maybe build a helper function that makes sure a response is a proper HTTP response object?